index.html

```diff
@@ -2,26 +2,25 @@
 <html lang="en">
     <head>
         <meta charset="utf-8">
-        <title>FOSSASIA 2017 Singapore | Asia's Open Technology Event, Open Hardware, Free Software, Open Design, Open Data</title>
+        <title>FOSSASIA 2016 Singapore | Asia's Open Technology Event, Open Hardware, Free Software, Open Design, Open Data</title>
```

Comment:
@jig08 I have just added links in footer in lines 572, 573 and 574... I haven't made any other change in the whole code..